public/index.html

```diff
@@ -201,12 +202,56 @@ <h4><a href="https://www.safehorizon.org/hotlines">Safe Horizon Crime Victim Hot
                 </p>
                 <p class="phones">
                     <a href="tel:1-800-621-4673">Domestic Violence: 1 (800) 621-4673</a>
-                    <a href="tel:1-866-689-4357">Crime Victims: 1 (866) 689-4357</a>
                     <a href="tel:1-212-227-3000">Rape and Sexual Assault: 1 (212) 227-3000</a>
                 </p>
             </div>
         </section>
 
+        <section>
+            <header>
+                <h3 id="legal">Legal</h3>
+                <div class="buttons"><a href="#alternative-resources">Back to the top</a></div>
+            </header>
+
+            <div class="resource">
+                <h4><a href="https://nlgnyc.org/">National Lawyer Guild Arrest Support Hotline</a></h4>
+                <p>
+                    The Mass Defense Committee (MDC) is a network of lawyers, legal workers and law students providing legal support for political activists, protesters and movements for social change.
+                </p>
+                <p>
+                    MDC members in chapters across the country provide trainings, assistance in setting up temporary legal offices and legal support structures, and materials for supporting activists engaged in mass protests.
+                </p>
+                <p class="phones">
+                    <a href="tel:2066587963"><img src="/phone.svg" alt="Call" /></a>
+                    <a href="tel:2066587963">Seattle Hotline: (206) 658-7963</a>
+                    <a href="tel:2126792811">National Hotline: (212) 679-2811</a>
+                </p>
+            </div>
+
+            <div class="resource">
+                <h4><a href="https://www.safehorizon.org/hotlines">Safe Horizon Crime Victim Hotline</a></h4>
+                <p>
+                    The Safe Horizon Crime Victim hotline advocates offer personalized, non-judgmental support. They can talk to you about your safety, provide information about resources in your community, and connect you to our programs and services.
+                </p>
+                <p>
+                    Their confidential hotlines provide one-on-one support including:
+                    <ul>
+                        <li>Crisis counseling</li>
+                        <li>Safety planning</li>
+                        <li>Assistance with finding Domestic Violence Shelters</li>
+                        <li>Information about resources</li>
+                    </ul>
+                </p>
+                <p>
+                    They can assist in any language.
+                </p>
+                <p class="phones">
+                    <a href="tel:18666894357"><img src="/phone.svg" alt="Call" /></a>
+                    <a href="tel:18666894357">1 (866) 689-4357</a>
+                </p>
+            </div>
```

Comment:
This one doesn't have a phone number so maybe it's not as useful, but we could also add Puget Sound Prison Support: https://twitter.com/PugetSupport

Reply:
Ah I wish they had a phone number we could list. Maybe we create a separate website for general resources and local bail funds?

Reply:
Fair, definitely not required if it's out of the scope of this project! Just thought it worth mentioning :slightly_smiling_face: